app/components/alchemy/ingredients/text_view.rb

```diff
@@ -21,8 +21,7 @@ def call
           link_to(value, url_for(link), {
             id: dom_id.presence,
             title: link_title,
-            target: ((link_target == "blank") ? "_blank" : nil),
-            data: {link_target: link_target}
+            target: link_target.present? ? "_#{link_target}" : nil
```

Comment:
🤔 we are adding and removing the underscore of the link target all the time. Could we try to just use the value as is and remove all the regex replacement and string interpolation?

Reply:
Done, it is now way simpler.